### ISSUE TITLE
Js error on editor when drag en drop... 

### ISSUE THREAD

**Issue by Joystick31** [elementor/elementor#12310]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

When i custom my page with elementor builder. Many action was broken when duplicate or drag and drop (and maybe other issues). Try to reproduce to see

## Steps to reproduce

- Open Page with elementor editor
- Create first block with many elements in
- Try to remove, an JS error say "Cannot set property children of #<Container> which has only a getter"
- This problem happen on all duplication/drag&drop between two columns
- Very frustrating because, force to need to save, and reload page to see modification. Sometimes when we duplicate a section and we drag in an other position of the page, the section crash and no change happend...

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server (GPL) v5.7.30
	PHP Version: 7.4.9
	PHP Max Input Vars: 1000
	PHP Max Post Size: 67108864
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.5
	Site URL: https://afr-conseils.nativeweb.fr
	Home URL: https://afr-conseils.nativeweb.fr
	WP Multisite: No
	Max Upload Size: 64 Mo
	Memory limit: 268435456
	Permalink Structure: /%postname%
	Language: fr-FR
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Custom Template Native Web
	Version: 1.1584074612
	Author: Native Web
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.3.0
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: fr_FR
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.135 Safari/537.36

== Active Plugins ==
	Complianz | GDPR/CCPA Cookie Consent
		Version: 4.7.0
		Author: Really Simple Plugins

	Crocoblock Wizard
		Version: 1.0.1
		Author: Crocoblock

	Duplicator
		Version: 1.3.36
		Author: Snap Creek

	Elementor
		Version: 3.0.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.0.1
		Author: Elementor.com

	Envato Elements
		Version: 2.0.5
		Author: Envato

	Essential Addons for Elementor
		Version: 4.2.2
		Author: WPDeveloper

	Essential Addons for Elementor - Pro
		Version: 4.1.3
		Author: WPDeveloper

	Imagify
		Version: 1.9.10
		Author: WP Media

	JetEngine
		Version: 2.4.11
		Author: Crocoblock

	JetWooBuilder For Elementor
		Version: 1.6.5
		Author: Crocoblock

	WP Maintenance Mode
		Version: 2.2.4
		Author: Designmodo


== Must-Use Plugins ==
	1&1 Product Subdomain
		Version: 1.1.0
		Author: 1&1


== Elements Usage ==
	
	footer : 1
		divider : 1
		heading : 3
		icon-list : 1
		nav-menu : 1
		text-editor : 1
		theme-site-logo : 1
	header : 1
		nav-menu : 1
		theme-site-logo : 1
	wp-page : 1
		button : 4
		divider : 4
		form : 1
		heading : 9
		icon : 1
		slides : 1
		text-editor : 6


== Log ==
	
Log: showing 20 of 202020-02-15 10:26:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.8.5',
  'to' => '2.9.1',
)]
2020-03-12 12:17:55 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.1',
  'to' => '2.9.6',
)]
2020-03-14 14:36:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.8.3',
  'to' => '2.8.5',
)]
2020-03-31 19:08:18 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.6',
  'to' => '2.9.7',
)]
2020-04-17 09:13:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.8.5',
  'to' => '2.9.2',
)]
2020-06-12 11:19:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.7',
  'to' => '2.9.11',
)]
2020-06-12 11:19:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.9.2',
  'to' => '2.10.0',
)]
2020-08-17 14:14:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.11',
  'to' => '2.9.14',
)]
2020-08-17 14:14:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.10.0',
  'to' => '2.10.3',
)]
2020-08-29 05:36:27 [info] elementor::elementor_updater Started 
2020-08-29 05:36:27 [info] Elementor/Upgrades - _v_3_0_0_move_general_settings_to_kit Start  
2020-08-29 05:36:27 [info] Elementor/Upgrades - _v_3_0_0_move_general_settings_to_kit Finished 
2020-08-29 05:36:27 [info] Elementor/Upgrades - _v_3_0_0_move_default_colors_to_kit Start  
2020-08-29 05:36:27 [info] Elementor/Upgrades - _v_3_0_0_move_default_colors_to_kit Finished 
2020-08-29 05:36:27 [info] Elementor/Upgrades - _v_3_0_0_move_saved_colors_to_kit Start  
2020-08-29 05:36:27 [info] Elementor/Upgrades - _v_3_0_0_move_saved_colors_to_kit Finished 
2020-08-29 05:36:27 [info] Elementor/Upgrades - _v_3_0_0_move_default_typography_to_kit Start  
2020-08-29 05:36:27 [info] Elementor/Upgrades - _v_3_0_0_move_default_typography_to_kit Finished 
2020-08-29 05:36:27 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.14',
  'to' => '3.0.3',
)]
2020-08-29 05:39:39 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.10.3',
  'to' => '3.0.1',
)]

PHP: showing 5 of 5PHP: 2020-03-14 07:29:48 [notice X 18][/homepages/10/d709422710/htdocs/clickandbuilds/Afr/wp-content/plugins/elementor/includes/elements/column.php::1020] Undefined index: _column_size [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-08-29 07:01:18 [notice X 29][/homepages/10/d709422710/htdocs/clickandbuilds/Afr/wp-content/plugins/elementor/includes/base/controls-stack.php::1430] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-08-29 07:01:49 [notice X 20][/homepages/10/d709422710/htdocs/clickandbuilds/Afr/wp-content/plugins/elementor/includes/managers/image.php::86] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-08-29 07:02:07 [notice X 4][/homepages/10/d709422710/htdocs/clickandbuilds/Afr/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-08-29 07:20:02 [error X 2][/homepages/10/d709422710/htdocs/clickandbuilds/Afr/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::281] Uncaught Error: Call to a member function update_meta() on bool in /homepages/10/d709422710/htdocs/clickandbuilds/Afr/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:281
Stack trace:
#0 /homepages/10/d709422710/htdocs/clickandbuilds/Afr/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(162): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /homepages/10/d709422710/htdocs/clickandbuilds/Afr/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /homepages/10/d709422710/htdocs/clickandbuilds/Afr/wp-includes/class-wp-hook.php(287): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /homepages/10/d709422 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 3 of 3JS: 2020-03-21 17:05:52 [error X 7][https://afr-conseils.nativeweb.fr/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.9.6:31:175083] Cannot read property \'get\' of undefined 
JS: 2020-03-21 17:14:21 [error X 18][https://afr-conseils.nativeweb.fr/wp-content/plugins/elementor/assets/js/editor-document.min.js?ver=2.9.6:2:8328] container or containers are required. 
JS: 2020-08-29 07:33:52 [error X 1][https://afr-conseils.nativeweb.fr/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.3:2:470394] Cannot read property \'document\' of undefined 


				
```
</details>

**Comment by askinne2** [comment]
I also have been experiencing this issue. It has made editing my pages with Elementor nearly impossible 👎

**Comment by deanna** [comment]
Do you all happen to use the HappyAddOns Pro plugin? I just opened a bug with them, as disabling the Pro HAO plugin fixes the issue... [https://github.com/weDevsOfficial/happy-elementor-addons/issues/57](https://github.com/weDevsOfficial/happy-elementor-addons/issues/57)

**Comment by Joystick31** [comment]
> Do you all happen to use the HappyAddOns Pro plugin? I just opened a bug with them, as disabling the Pro HAO plugin fixes the issue... [weDevsOfficial/happy-elementor-addons#57](https://github.com/weDevsOfficial/happy-elementor-addons/issues/57)

Not for me. I have other addons but i was just using the default plugin elementor. No complementary elementor addons...

**Comment by shilo-ey** [comment]
I merged this issue with #12242, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!
